### PR TITLE
Integration test for user deleting account

### PR DESCRIPTION
**Pull request by ethantkoenig**

### CODE REVIEW

File: integrations/integration_test.go [hunk 1]
```diff
@@ -1,4 +1,3 @@
-// Copyright 2017 The Gitea Authors. All rights reserved.
```
Comment (lafriks) [inline integrations/integration_test.go]: Please revert this change

**Review by lafriks — CHANGES_REQUESTED** [review]

File: integrations/integration_test.go (hunk 1 above)
Comment (ethantkoenig) [reply integrations/integration_test.go]: My mistake, good catch

**Review by lafriks — APPROVED** [review]

**Comment by lafriks** [comment]
LGTM

**Comment by lunny** [comment]
LGTM

**Pull request merged**